packages/ui/src/partials/react-form/form-description.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import { useFormField } from "./form-field"
+import { cn } from "../../lib/tw-class-merge"
+
+export const FormDescription = React.forwardRef<
+  HTMLParagraphElement,
+  React.HTMLAttributes<HTMLParagraphElement>
+>(({ className, ...props }, ref) => {
+  const { formDescriptionId } = useFormField()
+
+  return (
+    <p
+      ref={ref}
+      id={formDescriptionId}
+      className={cn("text-sm text-gray-400", className)}
```

Comment:
Can we move the UI to the partials and use only the react hook form business here. Let these files only have react hook form business. The UI can be transferred to the partials or some other relevant place.

Reply:
Yes, that can be done